### ISSUE TITLE
Completely different results depending on the location

### ISSUE THREAD

**Issue by przemoo83** [pat310/google-trends-api#147]
It completely blew my mind. I have the same piece of code on my `localhost` and remote server (Heroku). For the query with identical parameters I get completely different results depending on whether I run it locally or om Heroku server. The difference is not trivial, it can be something like `0 to 100`. What is going on here?

**Comment by nametviner** [comment]
I'm having the same issue. Were you able to find a way around it?

**Comment by przemoo83** [comment]
No. I think that's not this APIs fault but it is just the way Google Trends works. Maybe different regions are served by different data centers? Or that's just a little Google's trick to discourage you from using their data like this? :P I think no one knows the answer
